modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1510,8 +1587,8 @@ public void scrollToTop(int index) {
 
         if (! posSet) {
             adjustPositionToIndex(index);
-            double offset = - computeOffsetForCell(index);
-            adjustByPixelAmount(offset);
+//            double offset = - computeOffsetForCell(index);
+//            adjustByPixelAmount(offset);
```

Comment:
I forgot to ask about this last time. Is there a reason to leave the former, now commented-out, code here, as opposed to just removing it?

Reply:
no reason, corrected.